azure-pipelines.yml

```diff
@@ -1,12 +1,111 @@
-trigger:
-  tags:
-    include:
-    - v*.*.*
-
-steps:
-  - task: Npm@1
-    inputs:
-      command: 'publish'
-      workingDir: '.'
-      publishEndpoint: 'NPM Registry'
-      verbose: true
+# Azure DevOps pipeline to build, check source codes, run tests, and publish
```

Comment:
This is a pipeline format we're no longer using. See this as an example: https://github.com/pagopa/io-functions-commons/tree/master/.devops

In brief, we split the pipeline into two different scenarios (deploy and code review) and move everything into a .devops folder

Reply:
Sorry, I'm looking the wrong configuration from `io-functions-commons`

Reply:
Use new version from `.devops` folder in 368b192